data/json/mutations.json

```diff
@@ -1423,6 +1423,76 @@
         "armor" : [ { "parts" : "ALL", "bash" : 1 } ]
     },{
         "type" : "mutation",
+        "id" : "LAMIA_SCALES_AMORED",
+        "name" : "Armored Scales",
+        "points" : 10,
+        "visibility" : 15,
+        "ugliness" : 4,
+        "description" : "Your tail and mouth have grown bullet resistant scales",
+        "cancels" : ["SCALES", "FEATHERS", "CHITIN", "PLANTSKIN", "FELINE_FUR", "LUPINE_FUR", "CHITIN_FUR"],
+        "prereqs" : ["LAMIA_TAIL"],
+        "threshreq" : ["THRESH_LAMIA"],
+        "category" : ["MUTCAT_LAMIA"],
+        "armor" : [ 
+            { "parts" : "LEG_L", "bash" : 11, "cut" : 24 }, 
+            { "parts" : "LEG_R", "bash" : 11, "cut" : 24 },
+            { "parts" : "FOOT_L", "bash" : 11 , "cut" : 24 },
+            { "parts" : "FOOT_R", "bash" : 11 , "cut" : 24 },
+            { "parts" : "MOUTH", "cut" : 14}
+            ]
+    },{
+        "type" : "mutation",
+        "id" : "LAMIA_SCALES",
+        "name" : "Lamia Scales",
+        "points" : 4,
+        "visibility" : 10,
+        "ugliness" : 1,
+        "description" : "You have grown a smooth, scale-like skin over your legs and mouth. While luxuriously soft, they offer no real benefit",
+        "cancels" : ["SCALES", "FEATHERS", "CHITIN", "PLANTSKIN", "FELINE_FUR", "LUPINE_FUR", "CHITIN_FUR"],
+        "category" : ["MUTCAT_LAMIA"]
+    },{
+        "type" : "mutation",
+        "id" : "LAMIA_TAIL",
+        "name" : "Snake Tail",
+        "points" : 14,
+        "visibility" : 15,
+        "ugliness" : 6,
+        "description" : "Your legs now terminate in an enormous noodly tail. You can now move greatly faster.",
+        "cancels" : ["SCALES", "FEATHERS", "CHITIN", "PLANTSKIN", "FELINE_FUR", "LUPINE_FUR", "CHITIN_FUR"],
+        "prereqs" : ["LAMIA_TAIL_JEUVENILE"],
+        "threshreq" : ["THRESH_LAMIA"],
+        "category" : ["MUTCAT_LAMIA"],
+        "purifiable" : false,
+        "restricts_gear" : [ "FOOT_L", "FOOT_R", "LEG_L", "LEG_R"],
+        "passive_mods" : {
+            "str_mod" : 1,
+            "dex_mod" : 2
+        }
+    },{
+        "type" : "mutation",
+        "id" : "LAMIA_TAIL_JEUVENILE",
+        "name" : "Short Snake Tail",
+        "points" : 14,
+        "visibility" : 15,
+        "ugliness" : 6,
+        "description" : "Your legs have fused together to form the beginings of a snake tail. While inherently useless now, you hope it will grow much bigger",
```

Comment:
There is in this case. Taking a lamia mutagen through the IV forces you to sleep for awhile, so that you can process the much larger physiology changes. I mean you literally end up with a tail thats somewhere around 15 to 25 feet. And that's reflected in the mutagens side of this PR.